src/classes/UTIL_CascadeDeleteLookups_TDTM.cls

```diff
@@ -228,35 +230,64 @@ public abstract class UTIL_CascadeDeleteLookups_TDTM extends TDTM_Runnable {
                     }
                 }
 
-                String error = String.format(Label.CascadeDeletionError, new List<String>{parentName}) + '<br/><br/>';
-                for (String errType : cc.errors.get(parentId).keySet()) {
-                    error += ' ' + errType + ': ' + getErrorLinks(cc,cc.errors.get(parentId).get(errType));
+                List<String> errorTypeWithChildRecords = new List<String>();
+                for (String errorType : cc.errors.get(parentId).keySet()) {
+                    if (USER_UserService.isUserRunningLightning() || USER_UserService.isUserRunningMobile()) {
+                        errorTypeWithChildRecords.add(errorType + ' (' + getPlainTextChildRecords(cc.children, cc.errors.get(parentId).get(errorType)) + ')');
+                    }
+                    else {
+                        errorTypeWithChildRecords.add(errorType + ':' + getHTMLTextChildRecords(cc.children, cc.errors.get(parentId).get(errorType)));
+                    }
+                }
+
+                String errorMessage = '';
+                if (USER_UserService.isUserRunningLightning() || USER_UserService.isUserRunningMobile()) {
+                    errorMessage = String.format(Label.CascadeDeletionErrorLightning, new List<String>{parentName, String.join(errorTypeWithChildRecords, ',')});
+                }
+                else {
+                    errorMessage = String.format(Label.CascadeDeletionError, new List<String>{parentName, String.join(errorTypeWithChildRecords, '')});
                 }
-                //don't escape error message with HTML links
-                parent.addError(error, false);
+
+                parent.addError(errorMessage, false);
             }
         }
     }
 
     /*******************************************************************************************************
-    * @description Creates a list of the child records preventing deletion, with html links to those records
-    * with the record's name.
-    * @param cc The Cascade these records belong to.
-    * @param records The child records preventing deletion.
+    * @description Get a list of the child records preventing deletion, with html links to those records
```

Comment:
Add an 's' to "Get" in the description or possibly use the word "Returns".

Also, I like your description of the Plain Text method. Maybe update this method's description to  -- "Gets an HTML list of the child records preventing deletion, with links to those records." Thoughts?